server/google-cloud/.env.example

```
@@ -0,0 +1,13 @@
+BASE_URL=https://vcknots.example.com
+PORT=8080
+
+PRIVATE_KEY_PATH=../samples/certificate-openid-test/private_key_openid.pem
+CERTIFICATE_PATH=../samples/certificate-openid-test/certificate_openid.pem
+
+# When using Google, please specify the following environment variables.
+GOOGLE_PROJECT_ID=<your-project-id-here>
+GOOGLE_PROJECT_LOCATION=<your-project-location-here>
+FIREBASE_PRIVATE_KEY=<firebase-admin-service-account-private-key-here>
+FIREBASE_CLIENT_EMAIL=<firebase-admin-client-email-here>
+FIRESTORE_DATABASE_ID=<firestore-database-id-here> # if not specified, the (default) is used.
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FIRESTORE_DATABASE_ID key should go before the GOOGLE_PROJECT_ID key

(UnorderedKey)